bitrise.yml

```diff
@@ -1695,6 +1549,536 @@ workflows:
+  #   - custom-test-results-export@0:
+  #       inputs:
+  #       - search_pattern: "$BITRISE_SOURCE_DIR/xcodebuild.xcresult"
+  #       - test_name: UnitTestsKlar
+  #   - deploy-to-bitrise-io@2: {}
+  #   - slack@3:
+  #       run_if: ".IsBuildFailed"
+  #       inputs:
+  #       - channel: "#mobile-alerts-ios"
+  #       - message: "The build run the Klar testPlan: UnitTests"
+  #       - webhook_url: "$SLACK_WEBHOOK"
+
+  # focus_unit_test:
+  #   before_run:
+  #   - focus-pull-and-unzip-dependencies
+  #   steps:
+  #   - script@1.1:
+  #       title: Test without Building
+  #       inputs:
+  #       - content: |
+  #           #!/usr/bin/env bash
+  #           set -ex
+  #           ls ./Users/vagrant/git/focus-ios/Blockzilla.xcodeproj/
+  #           cp -r ./Users/vagrant/git/focus-ios/Blockzilla.xcodeproj/* .
+  #           mv ./Users/vagrant/git/* .
+  #           ls -la
+  #           echo "-- test-without-building --"
+  #           xcodebuild -resultBundlePath "xcodebuild.xcresult" -derivedDataPath "/Users/vagrant/git/DerivedData" -destination "platform=iOS Simulator,name=iPhone 15,OS=17.2" test-without-building -xctestrun "/Users/vagrant/git/DerivedData/Build/Products/Focus_UnitTests_iphonesimulator17.2-arm64.xctestrun"
+  #   - custom-test-results-export@0:
+  #       inputs:
+  #       - search_pattern: "$BITRISE_SOURCE_DIR/xcodebuild.xcresult"
+  #       - test_name: UnitTests
+  #   - deploy-to-bitrise-io@2: {}
+  #   - slack@3:
+  #       run_if: ".IsBuildFailed"
+  #       inputs:
+  #       - channel: "#mobile-alerts-ios"
+  #       - message: "The build run the Focus testPlan: UnitTests"
+  #       - webhook_url: "$SLACK_WEBHOOK"
+  
+  # # FOCUS UTILITIES WORKFLOWS
+  # focus-clone-and-build-dependencies:
+  #   description: Clones the repo and builds dependencies
+  #   steps:
+  #   - activate-ssh-key@4:
+  #       run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+  #   - git-clone@6.2: {}
+  #   - certificate-and-profile-installer@1: {}
+  #   - script@1:
+  #       inputs:
+  #       - content: |-
+  #           #!/usr/bin/env bash
+  #           ./checkout.sh
+  #       title: ContentBlockerGen
+
+  # focus-pull-and-unzip-dependencies:
+  #   description: Pulls and unzip the dependencis from previous stage 
+  #   steps:
+  #   - git::https://github.com/bitrise-steplib/bitrise-step-artifact-pull.git@main:
+  #       title: Pull artifacts
+  #       inputs:
+  #       - verbose: true
+  #       - artifact_sources: build_applications.focus_configure_build
+  #   - script@1.1:
+  #       title: Unzip
+  #       inputs:
+  #       - content: |
+  #           set -euxo pipefail
+  #           echo "$BITRISE_ARTIFACT_PATHS"
```

Comment:
Is this going to need the changes similar to firefox_pull_and_unzip-dependencies?

`derived_data=${BITRISE_ARTIFACT_PATHS##*|}`

Reply:
No, explained in Slack, the value we get in that variable depends on the artifacts that we generate in previous stage and that are passed to next one. 
Since we specify that in this line: https://github.com/mozilla-mobile/firefox-ios/blob/ec351373e181a1cf50235c4f6d1c1b3d9a3fcb0b/bitrise.yml#L222

I don't expect changes in this. 
What we could do is to try to have a way to get that derived_data folder path for both app and re-use this code. But it will be easier to implement once we see both apps running I guess